front-end/src/components/Post/PostGovernanceInfo/PostMotionInfo.tsx

```diff
@@ -91,8 +91,11 @@ const ProposalInfo = ({ preimage } : {preimage?: OnchainLinkMotionPreimageFragme
 							? <>
 								<h6>Arguments</h6>
 								{preimageArguments.map((element, index) => {
-									return <div className={'methodArguments'} key={index}>
-										<span key={index}>{element.name}: {element.value}</span>
+									return <div className={element.name !== 'account' ? 'methodArguments' : ''} key={index}>
+										{element.name === 'account'
+											? <AddressComponent address={element.value} key={index}/>
+											: <span key={index}>{element.name}: {element.value}</span>
+										}
```

Comment:
to avoid repetition and enhance readability
```suggestion
									const isAccountArgument = element.name === 'account';
									return <div className={isAccountArgument ? '' : 'methodArguments'} key={index}>
										{isAccountArgument
											? <AddressComponent address={element.value} key={index}/>
											: <span key={index}>{element.name}: {element.value}</span>
										}
```